spec/suites/map/MapSpec.js

```diff
@@ -2499,7 +2627,118 @@ describe('Map', () => {
 			const offset = new Point(10, 100);
 			const output = map.unproject(offset);
 			expect(output).to.be.nearLatLng(expectedOutput);
+		});
+	});
+
+	describe('#flyToBounds', () => {
+		it('throws if map is not set before', () => {
+			expect(() => {
+				map.flyToBounds();
+			}).to.throw();
+		});
+
+		it('throws if passed invalid bounds', () => {
+			expect(() => {
+				map.flyToBounds(0, 0);
+			}).to.throw();
+		});
+
+		it('doesn\'t fly if already in bounds', () => {
+			map.setView([0, 0]);
+			const bounds = new LatLngBounds([
+				[-1, -1],
+				[1, 1],
+			]);
+			const expectedCenter = latLng([0, 0]);
+			expect(map.flyToBounds(bounds)).to.equal(map);
+			expect(map.getCenter()).to.be.nearLatLng(expectedCenter);
+		});
+
+		it('flies to requested bounds and corresponding center (low zoom case)', function (done) {
+			this.timeout(10000);
```

Comment:
```suggestion
```

The tests should not need the 10 seconds. Please try to disable animation: `map.setView([0, 0], 0, {animate: false});`, `map.flyToBounds(bounds, {animate: false})`